docs/tutorials/single-dev-workflow.md

```diff
@@ -210,51 +214,58 @@ epinio app logs mysimpleapp
 epinio app logs --follow mysimpleapp
 ```
 
-### Create a new port-forward
+### Create a new `port-forward`
 
-As described above, Epinio creates a new `ingress route` for your
-application. The route is bound by default to the port `443`.
+As mentioned, Epinio creates a new `ingress route` for your application.
+Epinio binds the route to port `443` by default.
 
-However, you might need to test parts of your application using a different
-port. For these specific cases, you can run the following command:
+However, you might need to test parts of your application using different
+ports. For these cases, run the following command, for example:
 
 ```shell
 epinio app port-forward mysimpleapp 8080:8080
 ```
 
 :::tip
 
-You can specify only one port number. In that case, Epinio will open the port of both `local` and `remote` targets.
+<!--TODO:I need some help with this. One port number? You can only use one app
```

Comment:
@enrichman @andreas-kupries ?

Reply:
I believe that the `one port number` means that you can write

```
epinio app port-forward mysimpleapp 8080
```

and epinio will treat the single `8080` as `8080:8080`.

Reply:
@enrichman Can you confirm ?

Reply:
If so, then that means port_forwarding.md is wrong. It appears to state that 

`epinio app port-forward mysimpleapp 8080`

is 

`epinio app port-forward mysimpleapp 8080:random`

on line 18

Reply:
@jhkrug as @andreas-kupries said when a single port is used it will be used the same as local and remote.

Regarding the `port_forwarding.md` is probably not completely clear and it can be rephrased.  

You have three different options on how to prove the `LOCAL:REMOTE` ports:
- `8080` -> the same port will be used as `LOCAL` and `REMOTE` (equivalent to `8080:8080`)
- `3456:8080` -> specific ports will be used
- `:8080` -> a random port will be used as `LOCAL` (equivalent to `<random>:8080`)

In the last case this is an example of the output:

```
-> % epinio app port-forward sample :8080                 

🚢  Executing port forwarding
Namespace: workspace
Application: sample

Forwarding from 127.0.0.1:37677 -> 8080
Forwarding from [::1]:37677 -> 8080
Handling connection for 37677
```

As you can see a random available port (the `37677`) was selected from the host machine.